docs/hardware/raspberry-pi/makecode-wooden-cabinet.md

```diff
@@ -0,0 +1,76 @@
+# MakeCode Arcade Wooden Cabinet
+
+This guide is not complete and is meant as a live note for the cabinets built in Microsoft Building 99.
+
+### ~ hint
+
+#### Necessary equipment
+
+To complete this project, you will need access to power tools. 
+Make sure to follow all safety procedures or
+ask a professional for help.
+
+### ~
+
+## Materials
+
+The cabinet is built out of 3 - 8' x 4' plywood sheets of 1/2" thickness. 
+
+* **3** - 8' x 4' x 1/2" maple plywood sheets. Feel free to use other sheets based on your availability. 
+* **1** - arcade button set for 2 players (https://www.amazon.com/gp/product/B00WDQWK5M)
+* **1** - 22" monitor with HDMI input (https://www.amazon.com/gp/product/B072F3DL6P)
+
+Tools needed
+
+* jigsaw, router, table saw, impact driver, pin nail gun, wood glue, etc... The kind of tools you find in a wood shop.
+* **1/8"** wood drill bit
+* **1/2"** wood drill bit
+
+## Cut list
+
+All units are in inches and angle in degrees. Add 1/4" to the length when a miter is present, 1/2" when 2 miters are present.
+
+| Thickness | Width | Length   | Location | Miter
+| --------- | ----- | -------- | -------- | -------
+| 1/2       | 23    | 59 3/4   | K        | 22 1/2
+| 1/2       | 23    | 24 7/8   | B        | 0
+| 1/2       | 23    | 22 1/2   | A        | 0
+| 1/2       | 23    | 18 1/2   | F        | 0
+| 1/2       | 23    | 16 1/4   | E        | 0
+| 1/2       | 23    | 15 7/8   | J        | 22 1/2
+| 1/2       | 23    | 12 15/16 | I        | 22 1/2
+| 1/2       | 23    | 12       | G        | 31
+| 1/2       | 23    | 11 1/2   | C        | 25
+| 1/2       | 23    | 6 1/4    | H        | 31
+| 1/2       | 23    | 5        | D        | 25
+
+You will also need to cut out the 2 sides of the arcade cabinet.
+
+## Cut Instructions
+
+* start by cutting and mitering all the cuts set aside
+* cut out the 2 profiles of the arcade. Once you have a side, you can use it as a template to cut out the other one (using a router)
+
+## Assembly instructions
+
+* create 2 jigs that let you position boards 1/4" from the side
+* cut out various 1" x 1/2" strips needed to secure the boards.
```

Comment:
I used 'strips' here instead of 'slices'. y/n?

Reply:
Yeah I need to take lots of pics